desktop/apps/consignments2/components/step_marker/index.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import block from 'bem-cn'
+import { connect } from 'react-redux'
+
+function StepMarker ({ currentStep, steps }) {
+  const b = block('consignments2-step-marker')
+
+  return (
+    <div className={b()}>
+      <div className={b('steps')}>
+        {
+          steps.map((step, index) => {
+            const active = (index <= currentStep)
+            return (
+              <div className={b('step').state({ active })} key={step.label}>
```

Comment:
I'd recommend doing `b('step', { active: index <= currentStep })` to get a BEM modifier. Then you can style like:

```
&__dot
  // ...other styles
  &_active
    color purple-color
```

I'd save `state` for when needing to interface with a separate style library.